src/main/java/bowling/domain/frame/Frames.java

```diff
@@ -0,0 +1,44 @@
+package bowling.domain.frame;
+
+import bowling.domain.pin.Pin;
+import bowling.dto.StateDtos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Frames {
+    private final List<Frame> frames;
+
+    private Frames() {
+        this.frames = new ArrayList<>();
+
+        frames.add(NormalFrame.init());
+    }
+
+    public static Frames init() {
+        return new Frames();
+    }
+
+    public void downPins(final Pin downPins) {
+        Frame curFrame = getCurFrame();
+        
+        curFrame.downPins(downPins);
+        curFrame.appendFrame(frames);
+    }
+
+    public boolean isBowlingEnd() {
```

Comment:
볼링 프레임을 관리하는 일급컬렉션이므로
```suggestion
    public boolean isAllFrameEnd() {
```
등의 모든 프레임의 종류 유무를 확인하는 메소드 명이 어떨까요??

Reply:
그 편이 훨씬 낫겠네요 👍